### PR TITLE
[CONVERTER] Change custom mode name

### PR DESCRIPTION
**Pull request by gichan-jang**
- Changed custom mode name to custom-code for callback type.
- Add user data to the custom callback.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: gst/nnstreamer/tensor_converter/tensor_converter.c [hunk 1]
```diff
@@ -478,7 +478,7 @@ gst_tensor_converter_set_property (GObject * object, guint prop_id,
         break;
       }
 
-      if (g_ascii_strcasecmp (strv[0], "custom") == 0)
+      if (g_ascii_strcasecmp (strv[0], "custom-code") == 0)
         self->mode = _CONVERTER_MODE_CUSTOM;
```
Comment (jaeyun-jung) [inline gst/nnstreamer/tensor_converter/tensor_converter.c:482]: let's consider enum for future work,

_CONVERTER_MODE_CUSTOM_CODE / _CONVERTER_MODE_CUSTOM_SCRIPT

**Review by jaeyun-jung — APPROVED** [review]

File: gst/nnstreamer/tensor_converter/tensor_converter.c (hunk 1 above)
Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c:482]: Checked.
I will update it when `custom-script` is introduced.

Comment (dongju-chae) [inline gst/nnstreamer/tensor_converter/tensor_converter.c:482]: Don't need to change the corresponding enum?

Comment (gichan-jang) [reply gst/nnstreamer/tensor_converter/tensor_converter.c]: Updated.

**Review by dongju-chae — APPROVED** [review]
LGTM

**Pull request merged**